server/bundler/i18n-calypso-loader.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Webpack loader used to inject Webpack code into `i18n-calypso` module to
+ * load asyncronously the current locale.
```

Comment:
> Looking at the documentation maybe it's because of double quotes that are missing?

You are right, good catch! Now the chunk name is generated correctly :) I've committed the changes.

Reply:
> [...] Are the locale bundles correctly minified in that case?

I pushed a new commit with changes in the minifier that makes it traverse all `./public` subdirectories to ensure that locale files are minified.

I tested it and now it works correctly.